iphone/Classes/TiUIListView.m

```diff
@@ -1762,7 +1762,7 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
     }
     if (TiDimensionIsDip(height)) {
         return height.value;
-    } else if (TiDimensionIsAutoSize(height)) {
+    } else if (TiDimensionIsAutoSize(height) || TiDimensionIsUndefined(height)) {
```

Comment:
Mhh, I'd be really curious if checking for size and fill is a good idea. Pretty sure it breaks something else.